addons/message_survey/manifest.json

```diff
@@ -0,0 +1,37 @@
+{
+  "api_version": "0.1",
+  "id": "message_survey",
+  "name": "Survey Message",
+  "type": "message",
+  "conditions": {
+    "trigger_time": 2419200,
+    "javascript": "SetConditions.js"
```

Comment:
I think we don't need the javascript condition if we restore the staging_1h and staging_30m